src/dom.js

```diff
@@ -985,8 +985,11 @@ export function getVerticalScrollbarWidth(win) {
 export function dispatchCustomEvent(node, name, opt_data, opt_options) {
   const data = opt_data || {};
   // Constructors of events need to come from the correct window. Sigh.
-  const event = /** @type {?}*/ (node.ownerDocument.createEvent('Event'));
-  event.data = data;
+  const event = node.ownerDocument.createEvent('Event');
+
+  // Technically .data is not a property of Event.
```

Comment:
```javascript
/**
 * @typedef {{data: T}}
 * @extends Event
 * @template T
 */
let EventWithData;
```
might be worth defining since we use this pattern elsewhere